SampleApps/ReceiverDemoSwift/ReceiverDemoSwift-Bridging-Header.h

```diff
@@ -0,0 +1,5 @@
+#import <DeepLinkKit/DeepLinkKit.h>
```

Comment:
In order to expose the Objective-c based pod to the Swift code, you do have to add the pod's framework into your bridging header. Cocoapods did add the `use_frameworks!` in 0.36 which would mean you don't have to do this but I *believe* that only supports iOS 8 and above.

That said, no you wouldn't include this in the actual pod.